es6-modules/src/components/todo-app.js

```diff
@@ -0,0 +1,56 @@
+
+import createElement from '../lib/create-element';
+import noop from '../lib/noop';
```

Comment:
While this works you can combine this into 1 line because of the `index.js` file in the `lib` folder. For example

```JAVASCRIPT
import { createElement, noop } from '../lib';
```